gateway/src/test/kotlin/com/example/gateway/web/rest/PassControllerTest.kt

```diff
@@ -0,0 +1,151 @@
+package com.example.gateway.web.rest
+
+import com.example.gateway.PassManagingGatewayApplication
+import com.example.gateway.configuration.NatsClient
+import com.example.gateway.proto.PassDtoFixture.passDto
+import com.example.gateway.proto.PassDtoFixture.passFromDto
+import com.example.gateway.web.dto.PassDto
+import com.example.gateway.web.mapper.proto.pass.CreatePassResponseMapper.toCreatePassRequest
+import com.example.internal.NatsSubject.Pass.CANCEL
+import com.example.internal.NatsSubject.Pass.CREATE
+import com.example.internal.NatsSubject.Pass.DELETE_BY_ID
+import com.example.internal.NatsSubject.Pass.FIND_BY_ID
+import com.example.internal.NatsSubject.Pass.TRANSFER
+import com.example.passmanagersvc.input.reqreply.CancelPassResponse
+import com.example.passmanagersvc.input.reqreply.CreatePassResponse
+import com.example.passmanagersvc.input.reqreply.DeletePassByIdResponse
+import com.example.passmanagersvc.input.reqreply.FindPassByIdResponse
+import com.example.passmanagersvc.input.reqreply.TransferPassResponse
+import com.example.passmanagersvc.util.PassProtoFixture
+import com.example.passmanagersvc.util.PassProtoFixture.cancelPassRequest
+import com.example.passmanagersvc.util.PassProtoFixture.deletePassByIdRequest
+import com.example.passmanagersvc.util.PassProtoFixture.succesfulCancelPassResponse
+import com.example.passmanagersvc.util.PassProtoFixture.succesfulDeletePassByIdResponse
+import com.example.passmanagersvc.util.PassProtoFixture.successfulCreatePassResponse
+import com.example.passmanagersvc.util.PassProtoFixture.successfulTransferPassResponse
+import com.example.passmanagersvc.util.PassProtoFixture.transferPassRequest
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.every
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest
+import org.springframework.http.MediaType
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.web.reactive.server.WebTestClient
+import org.springframework.test.web.reactive.server.expectBody
+import reactor.kotlin.core.publisher.toMono
+
+@ContextConfiguration(classes = [PassManagingGatewayApplication::class])
+@WebFluxTest(PassController::class)
+@ActiveProfiles("test")
```

Comment:
Do you need this @ActiveProfiles("test") && @ContextConfiguration(classes = [PassManagingGatewayApplication::class])?

Reply:
Don`t actually need it